shard-operator/src/test/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceTest.java

```diff
@@ -55,12 +55,13 @@ public void setup() {
     @WithPrometheus
     public void testBridgesAreDeployed() throws JsonProcessingException, InterruptedException {
         List<BridgeDTO> bridgeDTOS = new ArrayList<>();
-        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-1", "myName-1", "myEndpoint", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
-        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-2", "myName-2", "myEndpoint", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
+        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-1", "myName-1", "myEndpoint/events", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
+        bridgeDTOS.add(new BridgeDTO("bridgesDeployed-2", "myName-2", "myEndpoint/events", TestSupport.CUSTOMER_ID, ManagedResourceStatus.ACCEPTED, TestSupport.KAFKA_CONNECTION_DTO));
         stubBridgesToDeployOrDelete(bridgeDTOS);
         stubBridgeUpdate();
         String expectedJsonUpdateProvisioningRequest =
-                String.format("{\"id\": \"bridgesDeployed-1\", \"name\": \"myName-1\", \"endpoint\": \"myEndpoint\", \"customerId\": \"%s\", \"status\": \"provisioning\"}", TestSupport.CUSTOMER_ID);
+                String.format("{\"id\": \"bridgesDeployed-1\", \"name\": \"myName-1\", \"endpoint\": \"myEndpoint/events\", \"customerId\": \"%s\", \"status\": \"provisioning\"}",
+                        TestSupport.CUSTOMER_ID);
         String expectedJsonUpdateAvailableRequest =
                 String.format("{\"id\": \"bridgesDeployed-1\", \"name\": \"myName-1\", \"endpoint\": \"http://192.168.2.49/ob-bridgesdeployed-1\", \"customerId\": \"%s\", \"status\": \"ready\"}",
```

Comment:
This is the second thing I'd like some help with.. the whole provisioning in the test seems a little bit "real code" and a little bit "mocked/test code". I would love to be able to assert that the expected endpoint contained `/events` but I could not find what I might need to mock/change while digging around.